README.md

```diff
@@ -113,8 +113,9 @@ $ docker run -p 8080:80 -e SERVER_NAME=myhost my-modsec
 
 | Name     | Description|
 | -------- | ------------------------------------------------------------------- |
-| ACCESSLOG | A string value indicating the location of the custom log file (Default: `/var/ log/apache2/access.log`) |
-| APACHE_ALWAYS_TLS_REDIRECT | A string value indicating if http should redirect to https (Allowed values: `on`, `off`. Default: `off`) | 
+| ACCESSLOG | A string value indicating the location of the custom log file (Default: `/var/log/apache2/access.log`) |
+| APACHE_ALWAYS_TLS_REDIRECT | A string value indicating if http should redirect to https (Allowed values: `on`, `off`. Default: `off`) |
+| APACHE_LOGFORMAT | A string value indicating the LogFormat that apache should use. Default: `'"%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\""'` (combined). Tip: use single quotes outside your double quoted format string. \ |
```

Comment:
`|` must be prohibited as part of the log format so that `sed` works.

Reply:
We use that all over the place. Do you mean in general? We can create a new issue and track it there.

Reply:
Yes, but in most cases the use of the delimiter is unlikely. In the case of the log format it is easy to imagine someone thinking that `|` would be a nice separator. That's why I think it should be mentioned here explicitly.

Reply:
Ok, added mention about this.